src/components/planets/FirstPlanet.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { IconPropsWithText } from '../../Contract';
+import styles from './assets/planets.module.scss';
+
+export default function FirstPlanet(props: IconPropsWithText) {
```

Comment:
I think it will be better to create 3 planets as icons and one component with props {text, icon} which will look similar to this but will display icon from prop